src/CMakeLists.txt

```diff
@@ -149,6 +149,10 @@ set(LLVM_LIBS_DEP
   demangle
   OrcJIT
   native
+  ARMCodeGen
+  ARMAsmParser
+  AArch64CodeGen
+  AArch64AsmParser
```

Comment:
If you go that way, then also add X86 here. Otherwise cross-compiles won't work the other way around.